src/main/java/Calculator/Interface/BaseCalculatorInterface.java

```diff
@@ -0,0 +1,9 @@
+package Calculator.Interface;
+
+public interface BaseCalculatorInterface {
+    public double addition(double a, double b);
```

Comment:
interface methods are default public, you can remove the public modifiers from the methods.

Reply:
Done